libs/npm/core/src/hooks/api-editor/validation/AdvancedScheduleSchema.ts

```diff
@@ -6,7 +6,7 @@ import { ScheduleSchema } from './ScheduleSchema';
 /**
  * Validation schema for data source advanced schedule.
  */
-export const AdvancedScheduleSchema: Yup.ObjectSchema<IScheduleModel> = ScheduleSchema.shape({
+export const AdvancedScheduleSchema: Yup.SchemaOf<IScheduleModel> = ScheduleSchema.shape({
```

Comment:
@jdtoombs please make sure you have the latest locally.  Then clear out your tno-core dist and npm-modules folder.  Then run yarn install.  

You're undoing the changes to support yup upgrade.  I ran into the same thing.  Still not clear what's going on.  Talk too @CelloLiu  to see what he did.  

Make sure these changes are valid.

Reply:
Will do!

Reply:
@Fosol, closing this. Seems although my local core was not properly updated hence the errors